chef/cookbooks/crowbar/templates/default/crowbar-evacuate-host.sh.erb

```diff
@@ -47,7 +47,7 @@ fi
 
 # FIXME: here we need checks for potential errors during the evacuation
 
-while nova list --host "$host" --fields host 2>/dev/null | grep "$host"; do
+while nova list --all-tenants --status active --host "$host" --fields host 2>/dev/null | grep "$host"; do
```

Comment:
This is actually used during the live migration, where instances are not shutting down, but moving to different host. (So it's maybe confusing to have it in this PR).

Anyway, your comment probably still applies, do you have any idea how to make the check better? We need to make sure that the live-migration was successfully finished and the $host in question does not have any instances running (it could have some not-running though).